servers/services/src/main/java/org/projectnessie/services/rest/TreeResourceWithAuthorizationChecks.java

```diff
@@ -100,7 +100,6 @@ public Reference getReferenceByName(String refName) throws NessieNotFoundExcepti
   @Override
   public Reference createReference(@Nullable String sourceRefName, Reference reference)
       throws NessieNotFoundException, NessieConflictException {
-    // TODO remove after Iceberg > 0.12
```

Comment:
what changed that this comment can be removed?

Reply:
We always default to the default branch now. Nothing specific to this PR though - slipped though.